lib/templates/plugin.js

```diff
@@ -16,7 +16,9 @@ class TikTok {
 
 export default (ctx, inject) => {
   const instance = new TikTok(<%= JSON.stringify(options) %>);
-  instance.enable();
+  if (!instance.options.disabled) {
```

Comment:
Can you put l18, 22, 23 and 24 in the if statement?
Thanks!

Reply:
Hmm, I don't think that would work as you would then be unable to call the `enable` function, which you would still want to do once consent has been given.

Reply:
You are right, thanks for the note, I am in between projects now, not focused enough ;)